libipt/CMakeLists.txt

```diff
@@ -83,7 +83,9 @@ endif (CMAKE_HOST_WIN32)
 
 set(LIBIPT_FILES ${LIBIPT_FILES} ${LIBIPT_SECTION_FILES})
 
-add_library(libipt SHARED
+option(BUILD_SHARED_LIBS "Build the shared library" ON)
+
+add_library(libipt
```

Comment:
there's another library we might want to build static or shared in sideband/.  it is currently not installed since man pages are still missing but if we're making this change, I think we should include that one, too.

you can move the option into the top-level CMakeLists.txt and group it together with the other options, there.